web_client/templates/panels/annotationSelector.pug

```diff
@@ -67,14 +67,16 @@ block content
                   download=name + '.json')
                 span.icon-download.h-download-annotation(
                     title='Download annotation')
-
+  
   .checkbox.h-annotation-toggle
     label(title='Show annotation labels when hovering with the mouse.')
       input#h-toggle-labels(type='checkbox', checked=showLabels)
       | Labels
     label(title='Highlight annotations when hovering with the mouse.')
       input#h-toggle-interactive(type='checkbox', checked=interactiveMode)
       | Interactive
-    button.btn.btn-sm.btn-primary.h-create-annotation
-      | #[span.icon-plus-squared] New
+    if user && user.get 
```

Comment:
I don't think we need to check if user is defined.  I think the accessLevel will be appropriate if no user is logged in.

Reply:
Ok.